app/controllers/requests_controller.rb

```diff
@@ -134,4 +135,79 @@ def request_params
     params.require(:request).permit(:name, :cpu_cores, :ram_mb, :storage_mb, :operating_system,
                                     :port, :application_name, :comment, :rejection_information, :status, user_ids: [], sudo_user_ids: [])
   end
+
+  # TODO get credentials and config for currently logged in user
```

Comment:
When will this be done?

Reply:
I will talk to the customer in 20 minutes about getting access to the repo, hopefully, the credentials will be available then. Please do not hard code in the credentials though, this is a public repo.
Use a .env file or a secret for this (see below).

Reply:
Credentials will be in an env file. I don't think you need the current_user